packages/loader/container-loader/src/containerContext.ts

```diff
@@ -79,63 +170,37 @@ export class ContainerContext implements IContainerContext, IProvideLayerCompatD
 		return loaderCompatDetailsForRuntime;
 	}
 
-	constructor(
-		public readonly options: ILoaderOptions,
-		public readonly scope: FluidObject,
-		public readonly baseSnapshot: ISnapshotTree | undefined,
-		private readonly _version: IVersion | undefined,
-		public readonly deltaManager: IDeltaManager<ISequencedDocumentMessage, IDocumentMessage>,
-		public readonly storage: IContainerStorageService,
-		public readonly quorum: IQuorumClients,
-		public readonly audience: IAudience,
-		public readonly loader: ILoader,
-		public readonly submitFn: (
-			type: MessageType,
-			contents: unknown,
-			batch: boolean,
-			appData: unknown,
-		) => number,
-		public readonly submitSummaryFn: (
-			summaryOp: ISummaryContent,
-			referenceSequenceNumber?: number,
-		) => number,
-		/**
-		 * @returns clientSequenceNumber of last message in a batch
-		 */
-		public readonly submitBatchFn: (
-			batch: IBatchMessage[],
-			referenceSequenceNumber?: number,
-		) => number,
+	constructor(private readonly config: IContainerContextConfig) {
```

Comment:
Do we need to store a reference to the config object if we're already storing all of its individual properties?